build/index.html

```diff
@@ -398,7 +398,7 @@ <h4 class="modal-title"><i class="fa fa-clone"></i> Summarized Restoration Asses
       </div><!-- /.modal-dialog -->
   </div>
 
-  <!-- Begin page content -->
+  <!-- Begin page content -->r

```

Comment:
Interesting! that 'r' has manifested itself in our Build files. Instead of changing the code here, I would re-run the build locally, and then push back to this branch.